campaignion_wizard/src/WebformStep.php

```diff
@@ -33,8 +33,42 @@ public function pageCallback() {
 
     // Build form for webform_template select box.
     if (module_exists('campaignion_action_template')) {
+      // Check if there are existing form submissions.
+      $submissions_exist = FALSE;
+      $sql = <<<SQL
+SELECT count(*) AS submissions
+FROM {webform_submissions}
+WHERE nid=:nid
+SQL;
```

Comment:
I think something like this would be better (no need to count them all).

```php
$has_submissions = (bool) db_select('webform_submissions', 's')
  ->fields('s', ['sid'])
  ->condition('nid', $nid)
  ->range(0, 1)
  ->execute()
  ->fetchField();
```

or without the using as much of the db-abstraction layer:

```php
$q = 'SELECT 1 FROM {webform_submissions} WHERE nid=:nid';
$has_submissions = db_query($q, [':nid' => $nid])->fetchField();
```